api/src/service/domain/workflow/workflowitem_create.ts

```diff
@@ -199,6 +203,16 @@ export async function createWorkflowitem(
     if (Result.isErr(result)) {
       return result;
     }
+    const { document } = result as WorkflowitemDocumentUploaded.Event;
+    // document should be private
+    if (minioEndPoint) {
+      await repository.uploadDocument(document);
+      const eventData = {...result, document: {...document, base64: "", url: hostPort}};
```

Comment:
@galethil What is the purpose of "reseting" base64 property?

Reply:
@m-janos base64 property contains the full content of uploaded file. We don't want to store file to multichain but to min.io. So I'm uploading file to min.io and clearing base64 so the file content is not stored to multichain, just the file metadata is stored.

Reply:
@galethil Ok, thanks for the explanation